packages/video_player/video_player/ios/Classes/FLTVideoPlayerPlugin.m

```diff
@@ -592,46 +580,46 @@ - (void)dispose:(FLTTextureMessage*)input error:(FlutterError**)error {
 }
 
 - (void)setLooping:(FLTLoopingMessage*)input error:(FlutterError**)error {
-  FLTVideoPlayer* player = _players[input.textureId];
-  [player setIsLooping:[input.isLooping boolValue]];
+  FLTVideoPlayer* player = self.playersByTextureId[input.textureId];
+  player.isLooping = input.isLooping.boolValue;
 }
 
 - (void)setVolume:(FLTVolumeMessage*)input error:(FlutterError**)error {
-  FLTVideoPlayer* player = _players[input.textureId];
-  [player setVolume:[input.volume doubleValue]];
+  FLTVideoPlayer* player = self.playersByTextureId[input.textureId];
+  [player setVolume:input.volume.doubleValue];
 }
 
 - (void)setPlaybackSpeed:(FLTPlaybackSpeedMessage*)input error:(FlutterError**)error {
-  FLTVideoPlayer* player = _players[input.textureId];
-  [player setPlaybackSpeed:[input.speed doubleValue]];
+  FLTVideoPlayer* player = self.playersByTextureId[input.textureId];
+  [player setPlaybackSpeed:input.speed.doubleValue];
 }
 
 - (void)play:(FLTTextureMessage*)input error:(FlutterError**)error {
-  FLTVideoPlayer* player = _players[input.textureId];
+  FLTVideoPlayer* player = self.playersByTextureId[input.textureId];
   [player play];
 }
 
 - (FLTPositionMessage*)position:(FLTTextureMessage*)input error:(FlutterError**)error {
-  FLTVideoPlayer* player = _players[input.textureId];
+  FLTVideoPlayer* player = self.playersByTextureId[input.textureId];
   FLTPositionMessage* result = [[FLTPositionMessage alloc] init];
   result.position = @([player position]);
   return result;
 }
 
 - (void)seekTo:(FLTPositionMessage*)input error:(FlutterError**)error {
-  FLTVideoPlayer* player = _players[input.textureId];
-  [player seekTo:[input.position intValue]];
-  [_registry textureFrameAvailable:input.textureId.intValue];
+  FLTVideoPlayer* player = self.playersByTextureId[input.textureId];
+  [player seekTo:input.position.intValue];
```

Comment:
I didn't scrub this file for dot notation, just updated a few spots where it's inconsistent within the same line.